pkg/transport/managed_transport.go

```diff
@@ -135,28 +136,25 @@ func (mt *ManagedTransport) Serve(readCh chan<- routing.Packet, done <-chan stru
 		}
 	}()
 
-	// Redial loop.
-	for {
-		select {
-		case <-mt.done:
-			return
+	r := netutil.NewRetrier(3*time.Second, 5, 2)
```

Comment:
I think it might be great if we could set up the values in the config.

Reply:
I think that will be great too. But which values are you referring to?

Reply:
@evanlinjin I mean args of `NewRetrier`, i.e. backoff, times and factor.

Reply:
@nkryuchkov  @evanlinjin will set the values in config.